azure-communication-ui/chat/src/main/java/com/azure/android/communication/ui/chat/presentation/ui/chat/components/AcsChatSendMessageButton.kt

```diff
@@ -0,0 +1,48 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+package com.azure.android.communication.ui.chat.presentation.ui.chat.components
+
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.res.painterResource
+import androidx.compose.ui.tooling.preview.Preview
+import com.azure.android.communication.ui.chat.R
+import androidx.compose.foundation.Image
+import androidx.compose.foundation.clickable
+import androidx.compose.foundation.layout.padding
+import androidx.compose.foundation.layout.size
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.layout.ContentScale
+import androidx.compose.ui.semantics.Role
+import androidx.compose.ui.semantics.contentDescription
+import androidx.compose.ui.semantics.role
+import androidx.compose.ui.semantics.semantics
+import androidx.compose.ui.unit.dp
+
+@Composable
+internal fun AcsChatSendMessageButton(
+    contentDescription: String,
+    onClick: () -> Unit = {}
+) {
+    val semantics = Modifier.semantics {
+        this.contentDescription = contentDescription
+        this.role = Role.Image
+    }
+    val painter = painterResource(id = R.drawable.azure_communication_ui_chat_ic_fluent_send_message_button_20_filled)
+    Image(
+        painter = painter,
+        contentDescription = contentDescription,
+        contentScale = ContentScale.Crop,
+        modifier = Modifier
```

Comment:
Use lambda modifiers where possible.

I.e.
.padding { 2.25.dp }

Similar to the .clickable and .semantics

Lambda modifiers are a bit more performant in compose, as it can skip recomposition of composables and push modifiers to the layout pass.

Reply:
Thx for the suggestion. I think we can merge this PR first then apply that lambda change with Muhammad's later PR.